content/src/cloud/troubleshooting/authentication-failed.md

```diff
@@ -0,0 +1,27 @@
+---
+zendesk:
+  article_id: 25836973464733
+  name: Can't log into Cloud. "Authentication failed", or "Unexpected error - software token MFA"
+  position: 7
+  labels: cloud, troubleshooting, mfa
+---
+
+### Symptom
+
+You are in Home Assistant and when trying to log into Home Assistant Cloud, you are getting a message _Authentication failed_, or something like _Unexpected error - software token MFA_. You can't log into Home Assistant Cloud.
+
+### Cause
+
+_Authentication failed_, or _Unexpected error - software token MFA_ indicate that the software version on your Home Assistant is too old to support MFA for Home Assistant Cloud.
```

Comment:
```suggestion
_"Authentication failed"_, with _"Unexpected error - software token MFA"_ indicate that the software version on your Home Assistant is too old to support MFA for Home Assistant Cloud.
```
There are several ways you can end up with "Authentication failed", and the cause/resolution here is for when that is accompanied with "Unexpected error - software token MFA"
The quotes IDK.

Reply:
ah, ok. thank you for the catch!